cves/CVE-2015-3982.yml

```diff
@@ -49,7 +49,9 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  The session flush method does not sychronized properly with the database, in which the session key gets set to a empty
```

Comment:
This sentence's grammar is off. It should be synchronize, not synchronized, and "which causes the session key to get set" makes a lot more sense than "in which the session key gets set". "persons" is also missing an apostrophe.